### DISCUSSION TITLE
Version 1.26.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#432] [Announcements]
- Note linking: you can now link note widgets to files to keep their content in sync
- Added home button as gesture trigger
- Added support for [dynamic clock icons](https://kvaesitso.mm20.de/docs/developer-guide/integrations/icon-packs#clock-icons) in icon packs
- Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.26.0'>Version 1.26.0</a>.</em>

**Comment by SergiGTAr** [comment]
Loving it!

**Comment by rahaaatul** [comment]
Would you add Home Button/Gesture to Search/App Drawer option?
![Screenshot_2023_0624_102429](https://github.com/MM2-0/Kvaesitso/assets/86149861/3ab852d1-9a66-47f3-939f-c09aedad8e2f)

**Reply by MM2-0** [reply]
Not as primary gesture, no. But you can assign "open search" to any gesture, so that you have two gestures that open the search.

**Reply by rahaaatul** [reply]
Perhaps an option to disable it? Because I'd like to open it with home gesture and assistant shortcut and use the swipe down for notifications and other gestures for other stuff.
